blockchain/vm/internaltx_tracer.go

```diff
@@ -0,0 +1,498 @@
+package vm
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/common/hexutil"
+	"math/big"
+	"strconv"
+	"time"
+)
+
+var errEvmExecutionReverted = errors.New("evm: execution reverted")
+var errExecutionReverted = errors.New("execution reverted")
+var errInternalFailure = errors.New("internal failure")
+var emptyAddr = common.Address{}
+
+// InternalTxTracer is a full blown transaction tracer that extracts and reports all
+// the internal calls made by a transaction, along with any useful information.
+// It is ported to golang from JS, specifically call_tracer.js
```

Comment:
The license comment is missing and this line is better at the license comment.

Reply:
I see. I will add the license statement.